src/wallet/load.cpp

```diff
@@ -65,8 +65,8 @@ bool VerifyWallets(interfaces::Chain& chain)
         const fs::path path = fs::absolute(wallet_file, GetWalletDir());
 
         if (!wallet_paths.insert(path).second) {
-            chain.initError(strprintf(_("Error loading wallet %s. Duplicate -wallet filename specified."), wallet_file));
-            return false;
+            chain.initWarning(strprintf(_("Ignoring duplicate -wallet filename %s."), wallet_file));
```

Comment:
4dd26ef6e65f09f187d947497f15ada80ed5e6f5

nit: The term "filename" seems a bit incorrect for the `-wallet` argument value:
```suggestion
            chain.initWarning(strprintf(_("Ignoring duplicate -wallet=%s argument."), wallet_file));
```